kubernetes/helm/pinot/templates/broker/service.yaml

```diff
@@ -21,18 +21,18 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ include "pinot.broker.fullname" . }}
+  annotations:
+    type: "service"
+  {{- if .Values.broker.service.gcpInternalLB }}
+    cloud.google.com/load-balancer-type: Internal
```

Comment:
@ChethanUK I know the PR is merged but do you think we could improve this by removing hardcoded cloud specific annotations in favour of a fully generic annotation map? Most helm chart supports that with something like this:

```
      annotations:     
      {{- with .Values.service.annotations }}
      {{- toYaml . | nindent 8 }}
      {{- end }}
```

That would extend the support LB support to all cloud providers.

Reply:
Ya, I was thinking more in sense of load balancer point of view :(
Sent a PR: https://github.com/apache/incubator-pinot/pull/6985